odoo/addons/base/models/ir_attachment.py

```diff
@@ -316,7 +316,10 @@ def _postprocess_contents(self, values):
                     if w > nw or h > nh:
                         img.resize(nw, nh)
```

Comment:
By the by, it looks like the resizing isn't even applied (`Image.resize` returns a new image).

Which I guess is sorta good news for animated gifs, because `Image.resize` doesn't understand multi-frame images, so it removes them all. For multi-frame images (animated gifs) we'd need to extract the individual frames, resize them, then recompose the gif.